src/scene/vcMedia.cpp

```diff
@@ -351,7 +351,7 @@ void vcMedia::Cleanup(vcState * /*pProgramState*/)
 
 void vcMedia::SetCameraPosition(vcState *pProgramState)
 {
-  pProgramState->camera.position = m_image.position;
+  pProgramState->pActiveViewport->camera.position = m_image.position;
```

Comment:
The base function from `vcSceneItem` sets all of the viewports, not just the active one.